src/patternfly/components/Table/table-nested.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> table newcontext table--id=table--id table--attribute=(concat table-nested--attribute ' id="' table--id '" aria-label="Nested table"') table--modifier="pf-m-compact" table--grid="true"}}
+{{#> table newcontext table--id=table--id table--attribute=(concat table-nested--attribute ' id="' table--id '" aria-label="Nested table"') table--modifier="pf-m-compact pf-m-no-border-rows" table--grid="true"}}
```

Comment:
What do you think about updating the border modifier to .pf-m-borderless-rows , .pf-m-no-border-rows sounds a little odd to me